charts/ndb-operator/values.yaml

```diff
@@ -4,7 +4,7 @@ image:
   repository: ghcr.io/nutanix-cloud-native/ndb-operator/controller
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "v0.0.5"
+  tag: "v0.0.6"
```

Comment:
best is to let it empty in the value file

Reply:
Done